walk-in-interview/src/main/java/com/google/job/data/PaymentFrequency.java

```diff
@@ -0,0 +1,35 @@
+package com.google.job.data;
+
+/** Enumeration that represents the payment frequency. */
+public enum PaymentFrequency {
+    HOURLY("HOURLY"),
+    WEEKLY("WEEKLY"),
+    MONTHLY("MONTHLY"),
+    YEARLY("YEARLY");
+
+    private final String frequencyId;
+
+    PaymentFrequency(String frequencyId) {
+        this.frequencyId = frequencyId;
+    }
+
+    /**
+     * Gets the id string.
+     *
+     * @return Id string.
```

Comment:
`@return` is superfluous. Prefer this instead:
```
/** Returns the stable id for the payment frequency. can be used to store in database. */

```

Reply:
Fixed